apps/train_loc/lib/train_loc/manager.ex

```diff
@@ -86,22 +86,19 @@ defmodule TrainLoc.Manager do
     state = schedule_timeout(state)
 
     for event <- events, event.event == "put" do
-      _ =
-        Logger.debug(fn ->
-          "#{__MODULE__}: received event - #{inspect(event)}"
-        end)
-
-      case ManagerEvent.from_string(event.data) do
-        {:ok, manager_event} ->
-          update_vehicles(manager_event, state)
-
-        {:error, reason} ->
-          _ =
-            Logger.error(fn ->
-              Logging.log_string("Manager Event Parsing Error", reason)
-            end)
-
-          :ok
+      Logger.debug(fn ->
```

Comment:
question: does Dialyzer no longer complain about the different return types here?

Reply:
Hmmm, different return types? Not quite sure what you mean. Dialyzer isn't complaining about anything though.

Reply:
It turns out I'm way out of date here!

In Elixir 1.9.4, `Logger.debug/2` was a function, returning an `:ok | {:error, reason}` tuple. Dialyzer would complain that we didn't match on the values, which is why you see `_ = Logger.debug()` in a lot of places in our code.

In Elixir 1.10, `Logger.debug/2` became a macro, always returning `:ok`. So no more need for the extra `_`!